### ISSUE TITLE
 suggestion add suport for  Unknown Structure Scan /Thargoid Surface Site

### ISSUE THREAD

**Issue by matheus1814818128** [njthomson/SrvSurvey#79]
add suporte for  Unknown Structure Scan /Thargoid Surface Site as in guardian things with types and subtypes 
follow a link of cannon types  https://canonn.science/codex/thargoid-site-survey/

**Comment by njthomson** [comment]
Thank you @matheus1814818128 for your suggestion, I've been actively pondering your idea. I'm assuming each site has an idea total layout, otherwise this will be super hard.

The initial problem will be orientating the map. Guardian sites have solid/straight buttresses to align against, whereas Thargoid sites are full of curving edges.

Then there's the map itself...

Sadly screensots cannot be used, - the game renders them with perspective, which makes it impossible to consistently track against. I found with Guardian sites the map needed to be accurate within ~2m across the whole site, which will be a challenge to produce at Thargoid sites.

However, we could proceed without a background map - some Guardian Structures still have no background.

If no background, what things would you like to see on a map?

**Comment by matheus1814818128** [comment]
for guidance canonn created a method to define a "north" of the structure that all the same type and subtypes are aligned using leviathans and the lack of them being "north" the first leviathan after the largest space without leviathans as this is consistent across all the structures  so to align you can use the top of the structure that has a little hole in the middle of the "petals" of the central structure looking at this "north" defined by the canonn to align the map. There are more detailed criteria in the Canonn article I sent in the first message

and lastly it would be interesting to have the thargoid link locations, the thargoid device, data points and maybe Meta-Alloys

and sorry English is not my first language

**Comment by matheus1814818128** [comment]
all planets apart from hip 22460 have only 1 structure for the planet and the types and sub types of all the known ones have already been mapped, the types and sub types if it helps I have a spreadsheet that I was revisiting all of them and updating the data with new and old data

**Comment by matheus1814818128** [comment]
and sorry and my first time using github

**Comment by njthomson** [comment]
No worries.

Did Thargoid sites shift lat/long co-ordinates with Odyssey?

If there's a concrete list of the sites, with correct lat/long co-ordinates and an establish methodology for measuring the heading ... then this all becomes viable.

**Comment by matheus1814818128** [comment]
Yes, they changed the coordinates on Odyssey, some friends and I are going to each one and reviewing the data and updating the coordinates. If you want, I can send you the Google spreadsheet that we are using to write down the data.

**Comment by njthomson** [comment]
I saw someone asking about this on the Canonn Discord recently, was that you?

**Comment by matheus1814818128** [comment]
No, but it could be one of my friends who is also reviewing the data with me

**Comment by njthomson** [comment]
I'm going to let this suggestion go. It is still viable, but I will need some help. Let me know if you are still interested.
